core/chains/evm/logpoller/orm.go

```diff
@@ -7,223 +7,232 @@ import (
 	"math/big"
 	"time"
 
+	"github.com/lib/pq"
+
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/jmoiron/sqlx"
 	"github.com/pkg/errors"
 
 	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink-common/pkg/sqlutil"
 
 	ubig "github.com/smartcontractkit/chainlink/v2/core/chains/evm/utils/big"
-	"github.com/smartcontractkit/chainlink/v2/core/services/pg"
 )
 
+// TODO: Set a reasonable timeout
+const defaultTimeout = 10 * time.Second
```

Comment:
Just noting another TODO here

Reply:
Yeah we still need to figure out how we're going to deal with all the timeouts on queries. Was discussing this with Jordan a little bit. There's still more work to do on this PR, just wanted to open it for some discussions!

Reply:
Oh gotcha! I think I ran into this problem some place else. Let me see if I can find what we decided there.

Reply:
Actually realized it might not help here. I was thinking about this [timeout](https://github.com/smartcontractkit/chainlink/blob/f7865e840c8f05341873a097fe7cc7077bc73a97/common/client/node.go#L21) in common but I was using it for an RPC call which would be different from an ORM timeout.